content/en/docs/concepts/policy/limit-range.md

```diff
@@ -1,64 +1,61 @@
 ---
 reviewers:
 - sftim
-title: LimitRange
+- kbhawkey
+title: Limit Ranges
 content_template: templates/concept
 weight: 10
 ---
 
 {{% capture overview %}}
 
-By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on kubernetes cluster.
+By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on a Kubernetes cluster.
 With Resource quotas, cluster administrators can restrict the resource consumption and creation on a namespace basis.
-Within a namespace, a Pod/Container can still consume as much CPU and memory as the defined namespace's resource quota.
-there is a concern that one Pod/Container could monopolise its namespace resources.
+Within a namespace, a Pod or Container can still consume as much CPU and memory as the defined namespace's resource quota.
+there is a concern that one Pod or Container could monopolise its namespace resources.
 LimitRange is a tool to address this concern.
 
 {{% /capture %}}
 
 
 {{% capture body %}}
 
-A limit range, defined by a `LimitRange` object, provides constraints that can :
-- Enforce min/max compute resources usage per Pod/Container in a namespace
-- Enforce min/max storage request per PersistentVolumeClaim in a namespace
+A limit range, defined by a `LimitRange` object, provides constraints that can:
+- Enforce minimun and maximum compute resources usage per Pod or Container in a namespace

```

Comment:
Is the list rendering correctly? You probably need to `Add an extra line after line 22 and check the preview`.

Link to preview:
https://deploy-preview-14086--kubernetes-io-master-staging.netlify.com/docs/concepts/policy/limit-range/

Reply:
Fixed the list display